client/db/bolt/upgrades.go

```diff
@@ -197,7 +209,7 @@ func reloadMatchProofs(tx *bbolt.Tx) error {
 		if mBkt == nil {
 			return fmt.Errorf("match %x bucket is not a bucket", k)
 		}
-		proofB := getCopy(mBkt, proofKey)
+		proofB := mBkt.Get(proofKey)
```

Comment:
In `reloadMatchProofs`, the proof bytes are decoded, reencoded, and re-saved inside the transaction, so also safe to elide the copy.